src/routes/postSkipSegments.js

```diff
@@ -97,19 +98,74 @@ function sendWebhooks(userID, videoID, UUID, segmentInfo) {
     }
 }
 
-// submission: {videoID, startTime, endTime}
+function sendWebhooksNB(userID, videoID, UUID, startTime, endTime, category, probability, ytData) {
+    //let submissionInfoRow = db.prepare('get', "SELECT s.videoID, s.userID, s.startTime, s.endTime, s.category, u.userName, " +
+    //    "(select count(1) from sponsorTimes where userID = s.userID) count, " +
+    //    "(select count(1) from sponsorTimes where userID = s.userID and votes <= -2) disregarded " +
+    //    "FROM sponsorTimes s left join userNames u on s.userID = u.userID where s.userId=?",
+    //[userID]);
+    let submissionCount = db.prepare('get', "SELECT COUNT(*) count FROM sponsorTimes WHERE userID=?", [userID]);
+    let disregardedCount = db.prepare('get', "SELECT COUNT(*) disregarded FROM sponsorTimes WHERE userID=? and votes <= -2", [userID]);
+    let uName = db.prepare('get', "SELECT userName FROM userNames WHERE userID=?", [userID]);
+
+    let submittedBy = "";
+    try {
+        // If a userName was created then show both
+        if (uName.userName !== userID){
```

Comment:
```suggestion
        if (uName !== undefined && uName.userName !== userID){
```

Then, you can remove the try catch

Reply:
Reworked the query to always return something.